### DISCUSSION TITLE
coding interactive presentations in egui

### DISCUSSION THREAD

**Discussion by m172021** [emilk/egui#4618] [General]
Suppose we want to build a slideshow presentation. The common approaches are:

MS PowerPoint / Google Office -> WYSIWYG
LaTeX Beamer -> nice math, more technical
SVG + JS -> more interactive

Now, suppose the topic of the presentation is "a piece of software written in Rust/wasm32/egui".

We could build a slideshow, with screenshots + videos --- of we could EMBED THE PRESENTATION IN THE APP ITSELF.

So I'm curious about opening up a egui::Window .. adding some buttons, and building the presentation INTO THE APP ITSELF, so it interacts well with the rest of the app.

I am wondering if others have done this / related examples / etc ...

Thanks!

**Comment by YgorSouza** [comment]
See <https://github.com/emilk/egui_presentation>.
